Gifski/Constants.swift

```diff
@@ -5,6 +5,7 @@ struct Constants {
 	static let defaultWindowSize = CGSize(width: 360, height: 240)
 	static let backgroundImage = NSImage(named: "BackgroundImage")!
 	static let allowedFrameRate = 5.0...50.0
+	static let repeatCountRange = 0...100
```

Comment:
```suggestion
	static let loopCountRange = 0...100
```

Reply:
the name here is a preview of the changes to come to the upstream gifski library that I'm working on.  I'll make the change to the code here now, as it's more reflective of the current app's code.